account_vat_on_payment/__openerp__.py

```diff
@@ -73,6 +55,16 @@
         'test/account_invoice_7.yml',
         'test/account_invoice_8.yml',
         'test/account_invoice_9.yml',
+        'test/account_invoice_1_real.yml',
+        'test/account_invoice_2_real.yml',
+        'test/account_invoice_3_real.yml',
+        'test/account_invoice_4_real.yml',
+        'test/account_invoice_5_real.yml',
+        'test/account_invoice_6_real.yml',
+        'test/account_invoice_7_real.yml',
+        'test/account_invoice_8_real.yml',
+        'test/account_invoice_1_real_same_account.yml',
     ],
+    'images': [],
```

Comment:
It should be raised only a warnig, see https://github.com/OCA/maintainer-quality-tools/pull/298#issuecomment-159457373 as reference.